unskript-ctl/unskript_ctl_upload_session_logs.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2023 unSkript.com
+# All rights reserved.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE
+#
+#
+import os
+import shutil
+import tarfile
+import logging
+from datetime import datetime
+import requests
+import subprocess
+
+
+source_directory = '/var/unskript/sessions/logs'
+destination_directory = '/var/unskript/sessions/uploads'
+tar_file_path = '/var/unskript/sessions/session_logs.tgz'
+command = f'find {destination_directory} -type f -exec stat -c "%W %n" {{}} \\;'
+
+logging.basicConfig(filename="/var/log/unskript/upload_script.log",
```

Comment:
Use capital letters for globals. Also, use a global for /var/log/unskript/upload_script.log